glean-core/Cargo.toml

```diff
@@ -30,7 +30,7 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 serde = { version = "1.0.104", features = ["derive"] }
 serde_json = "1.0.44"
-rkv = { version = "0.18.4", default-features = false, features = ["lmdb"] }
+rkv = { version = "0.19.0", default-features = false, features = ["lmdb"] }
```

Comment:
Does Glean still need lmdb migration path, btw?

Reply:
probably not, but I need to write the doc to inform people still. We can do that in a followup.